modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/index.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+/**
+ * Base components exposure to Liferay module dynamic load-up
+ */
+
+export { default as Autocomplete } from './autocomplete/entry';
+export { default as DropdownMenu } from './dropdown/entry';
+export { default as Gallery } from './gallery/entry';
+export { default as ItemFinder } from './item_finder/entry';
+export { default as MiniCart } from './mini_cart/entry';
+export { default as QuantitySelector } from './quantity_selector/entry';
+export { default as SidePanel } from './side_panel/entry';
+export { default as StepTracker } from './step_tracker/entry';
+export { default as Summary } from './summary/entry';
+
+/**
+ * Components' contexts exposure to Liferay module dynamic load-up
+ */
+
+export { default as MiniCartContext } from './mini_cart/MiniCartContext';
```

Comment:
@marco-leo @FabioDiegoMastrorilli @garofalog

For sure I need the `MiniCartContext` for SaaS, but again many of these components are exposed to JSP's (and still reachable by direct path, even without this interface exposed). Might be worth considering invoking such components the proper way as suggested by FE Infra.

Could you please evaluate the list?